### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ssssota**
Very interesting project!

Fixed typo in the documents.

### CODE REVIEW

File: docs/docs/intro.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ sidebar_position: 1
 
 # What's Wasm Workers Server?
 
-Wasm Workers Server is a project that allows you to run serverless code using a lightweight construct called "workers", explained later in the document. Ther server itself is implemented as a self-contained binary that routes HTTP requests to a WebAssembly runtime that hosts the workers. It looks for `.wasm` and other compatible modules (like JavaScript files) in the given folder and associate HTTP endpoints based on their path:
+Wasm Workers Server is a project that allows you to run serverless code using a lightweight construct called "workers", explained later in the document. There server itself is implemented as a self-contained binary that routes HTTP requests to a WebAssembly runtime that hosts the workers. It looks for `.wasm` and other compatible modules (like JavaScript files) in the given folder and associate HTTP endpoints based on their path:
```
Comment (ereslibre) [inline docs/docs/intro.md:7]:
```suggestion
Wasm Workers Server is a project that allows you to run serverless code using a lightweight construct called "workers", explained later in the document. The server itself is implemented as a self-contained binary that routes HTTP requests to a WebAssembly runtime that hosts the workers. It looks for `.wasm` and other compatible modules (like JavaScript files) in the given folder and associate HTTP endpoints based on their path:
```

File: docs/docs/intro.md [hunk 2]
```diff
@@ -50,4 +50,4 @@ Wasm Workers Server is a lightweight implementation of a Worker platform that ai
 
 It aims for compatibility and follows an ongoing specification that different companies are working under the name of [WinterCG](https://wintercg.org/faq). This working group aims to create a common API for using Web Platform APIs like workers outside of the browser.
 
-Many of the platforms mentioned earlier follow a similar approach, so any code you write for Wasm Workers Server can be moved to those platforms easily (or the other way around!). Remember that our focus with wws is simplicity and compatibility. Since this is a growing ecosystem, we want you to start quickly and move wherever you need.
\ No newline at end of file
+Many of the platforms mentioned earlier follow a similar approach, so any code you write for Wasm Workers Server can be moved to those platforms easily (or the other way around!). Remember that our focus with wws is simplicity and compatibility. Since this is a growing ecosystem, we want you to start quickly and move wherever you need.
```
Comment (ereslibre) [inline docs/docs/intro.md]: I think we can remove this from the patch, I cannot see any difference.

**Review by ereslibre — CHANGES_REQUESTED** [review]
Thank you @ssssota! A couple of suggestions. Once that the license agreement is accepted we can merge your fixes!

File: docs/docs/intro.md (hunk 2 above)
Comment (ssssota) [reply docs/docs/intro.md]: I checked locally and found that line breaks were added.
I have pushed it again with that correction as well, so please check!

**Comment by gzurl** [comment]
Thanks @ssssota for your patch!
@ereslibre this LGTM now. But you are the change request owner :-)

**Review by gzurl — APPROVED** [review]
LGTM!

**Review by ereslibre — APPROVED** [review]

**Comment by ereslibre** [comment]
Thank you @ssssota!

**Pull request merged**